docs/get-started/index.md

```diff
@@ -15,5 +15,8 @@ Continue to:
 - **[Introduction](introduction.md)** for more details on the
   project.
 
-- **[Quick start](quick-start.md)** for quick step-by-step
-  instructions on how to get NFD running on your cluster.
+- **[Deployment](deployment/index.md)** for instructions on how to
+  deploy NFD-Operator to a cluster.
+
+- **[Advanced](advanced/index.md)** for more advanced topics and
+  reference.
```

Comment:
These links don't work. But as said. I suggest to rename get-started to introduction and rename introduction.md to index.md